synapse/storage/monthly_active_users.py

```diff
@@ -191,6 +190,7 @@ def user_last_seen_monthly_active(self, user_id):
             allow_none=True,
             desc="user_last_seen_monthly_active",
         ))
+        defer.returnValue(res)
```

Comment:
What's the reason for converting all these functions to defer.inlineCallbacks?

Reply:
me trying things and inadvertently committing it :) -- reverted